doc/rvv-intrinsic-spec.adoc

```diff
@@ -464,19 +555,42 @@ The intrinsics provide extra utility functions to help users manipulate across t
 
 == Programming Notes
 
+=== The `vd` parameters in the intrinsics
+
+Intrinsics whose computation is relevant to value held in `vd` have a passthrough operand in them. The following list enumerates the intrinsics that has a passthrough operand. Please see the appendix for the exact prototypes of these intrinsics.
+
+- Intrinsics with tail-undisturbed (`vta=0`)
+- Intrinsics with mask-undisturbed (`vma=0`)
+- Intrinsics representing Vector Multiply-Add Operations ^13^
+- Intrinsics representing Vector Slideup Instructions ^24^
+
+For intrinsics with no `vd` operand, the compiler is free to pick any register as the destination register.
+
+=== Assumption of `vstart=0` for intrinsics users.
+
+As the v-spec has mentioned:
+
+> The vstart CSR is writable by unprivileged code, but non-zero vstart values may cause vector instructions to run substantially slower on some implementations, so vstart should not be used by application programmers.
+
+The RVV intrinsics are designed for application users. The intrinsics will perform the semantic of vector instructions under `vstart=0`.
```

Comment:
Personally, I think portability (not performance) is a stronger rationale to hide `vstart` from the intrinsics API.

```suggestion

The `vstart` CSR is currently not exposed to the intrinsics programmer, 
and the intrinsics have the semantics of `vstart = 0`.
Support for positive `vstart` values is implementation defined,
thus portable application software should not set `vstart > 0`.
```

Reply:
Good point. Updated the note, thank you.